secure_message/common/utilities.py

```diff
@@ -31,8 +30,6 @@ def get_options(args, draft_only=False):
 
     if args.get('desc') and args.get('desc') == 'false':
         fields['desc'] = False
```

Comment:
Why is this doing 'and' on the same function call, could do

fields['desc'] = bool(strtobool(args.get('desc'))